source/core/utilities.cc

```diff
@@ -291,7 +291,7 @@ fill_vectors_from_file(std::map<std::string, std::vector<double>> &map,
           line_of_data = Utilities::string_to_double(list_of_words_clean);
           for (unsigned int i = 0; i < line_of_data.size(); ++i)
             {
-              map[column_names[i]].push_back(line_of_data[i]);
+              map[column_names[i]].emplace_back(line_of_data[i]);
```

Comment:
What is the advantage of using emplace_back here?

Reply:
Like we discussed :) prevents the generation of a copy.